milner_desvars/gazebo.md

```diff
@@ -0,0 +1,81 @@
+# Notes
+
+## Client and server separation
+
+When you run Gazebo you are running to different executables. In one hand 'gzserver' and in the other 'gzclient'.
+
+The 'gzserver' executable runs the physics update-loop and sensor data generation. This is the core of Gazebo and it can be used independently of a graphical interface. 
```

Comment:
Replace single quotes with inverted quotes to use [code formatting](https://help.github.com/en/articles/basic-writing-and-formatting-syntax#quoting-code).

Reply:
Apply this all over the text.